packages/angular_devkit/schematics_cli/README.md

```diff
@@ -0,0 +1,49 @@
+# Schematics CLI
+
+This package contains the executable for running a [Schematic](https://www.npmjs.com/package/@angular-devkit/schematics).
```

Comment:
Should this link to the NPM package or readme within Github https://github.com/angular/angular-cli/tree/master/packages/angular_devkit/schematics?

Reply:
I think GitHub is better, the link has been updated.

Reply:
Maybe relative is better?